libraries/AP_Common/Bitmask.h

```diff
@@ -35,6 +35,13 @@ class Bitmask {
         bits[word] |= (1U << ofs);
     }
 
+    // set all bits
+    void setall(void) {
+        for (uint16_t i=0; i<numwords; i++) {
+            bits[i] = 0xffffffff;
```

Comment:
This is incorrect for the last word, setting all the bits could result in the unused bits of the last word (say we requested 50 bits, setting the last 14 bits is invalid). This only really matters because you wanted to calculate the hamming weight on a bitmask which this will artificially throw off.